umbraco-heartcore/release-notes/2024-04-releasenotes.md

```diff
@@ -0,0 +1,13 @@
+# April 2024
+The following changes have been released to Heartcore sites created on or after February 12th. They will be rolled out to previously existing sites in the second quarter of 2024.
```

Comment:
Just to be sure @jmf-umbraco, is this sentence `The following changes have been released to Heartcore sites created on or after February 12th.` correct? or should it be a date here in April? 😊

Reply:
Yeah it's correct, but if you can think of a better way to say it, then feel free to revise.

For context - beneath the hood there are two versions of Heartcore sites. Feb 12th was when we cutover from creating older ones to newer ones. We have a migration in the pipeline to migrate the older ones, which aren't currently getting the fixes and feature updates that the newer ones are.